contributing.md

```diff
@@ -0,0 +1,59 @@
+# Contributing to {{Project-Name}}
+
+We welcome everyone to contribute what they can. Whether you are brand new, just want to contribute a little bit, or want to contribute a lot there is probably something you can help out with.
+
+## Where to get help
+The main way to get help is on our [discord server](https://discord.gg/uh69TdKfBD).  Though, this guide should help you get started. It may be slightly lengthy, but it's designed for those who are new so please don't let length intimidate you.
+
+## Code of Conduct
+Please be nice when interacting with others and follow our [code of conduct](<link to code of conduct>).
+
+## How to contribute
+If the code adds a feature that is not already present in an issue, you can create a new issue for the feature and add the pull request to it. If the code adds a feature that is not already present in an issue, you can create a new issue for the feature and add the pull request to it.
```

Comment:
Bring up the difference between GH issue and GH discussion here? (w/ links), right @bgins?

Reply:
Maybe it's:

```
### Contributing By Adding an Issue
...
### Contributing by Adding a Topic for Discussion?
```

Reply:
Ah yeah, that's right! We were going to add a bit about discussions vs issues. 💯 

I think that roughly speaking issues are for when you have a clearly defined bug report or feature request. Anything else probably falls under discussions. Examples might include:

- Community announcements and asking the community for feedback
- Discussing a new release
- Asking questions, Q&A that isn't for sure a bug report

There are some other ones listed in this post: https://resources.github.com/devops/process/planning/discussions/

One goal here is keeping issues focused on work that can be tied to PR. That way issues don't linger and stay open for extended periods of time.